Added Ownership of mongod.conf To MongoDB User

ddaskal opened the pull request:
Also fixed ceilometer service type in endpoints role.

j2sol commented on roles/endpoints/defaults/main.yml:
```diff
@@ -105,50 +105,50 @@ endpoints:
+      backend_api: 6384
       haproxy_api: 6385
   swift:
     name: swift
```
all the changes in this file seem unrelated. Please make this a different commit/pr

ddaskal commented:
Atom automatically removed all blank spaces after the last character for ever line when I saved the file. I only made one change per file:
1. Added 'owner=mongodb' for the mongod.conf file
2. Line 133, changed ceilometer 'type=metering' instead of 'type=Telemetry', to match the keystone service.

j2sol commented:
retest

ddaskal commented:
retest
